queries/fetch_latest_block.sql

```diff
@@ -0,0 +1,14 @@
+SELECT 
```

Comment:
Why is this query needed? Couldn't we simply use `u64::max` for `end_block` if we want all data until now? Also in terms of performance we probably need some kind of pagination mechanism (but I'm not sure).